### PR TITLE
Kpi balance

### PR DESCRIPTION
**Pull request by Leon-Chin**
this is the kpi and balanced board page

### CODE REVIEW

**Comment by Leon-Chin** [comment]
already resolved the conflicts

**Review by sulaimanmhashim — APPROVED** [review]

File: src/pages/Balanceboard.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                        rtkag.push(item['rtk-ag'])
+                    }
+                    return itemYear === currentYear && itemMonth === currentMonth
+                default:
+                    break;
+            }
+        })
+        setxAxis(xArr)
+        setPcr(pcr)
+        setRtkag(rtkag)
```
Comment (omar-al-hendi) [inline src/pages/Balanceboard.tsx:64]: Hi Lei, you could put this in the loader function and return xArr, pcr, rtkag as an object like this `{ xArr, pcr, rtkag }`

File: src/pages/Balanceboard.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    }
+    useEffect(() => {
+        getData()
+    }, [])
```
Comment (omar-al-hendi) [inline src/pages/Balanceboard.tsx:68]: if you use the loader we would not need to use these too

File: src/pages/KpiDashboard.tsx [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                default:
+                    break;
+            }
+        })
+        setxAxis(xArr)
+        setyAxis(yArr)
+    }
+    useEffect(() => {
+        getData()
+    }, [])
```
Comment (omar-al-hendi) [inline src/pages/KpiDashboard.tsx:63]: Same here bro, I suggest putting it in a loader function so we get better performance

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,2742 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (omar-al-hendi) [inline yarn.lock:4]: Aslo please delete the `yarn.lock` file we are using npm

**Review by omar-al-hendi — CHANGES_REQUESTED** [review]

**Comment by HeLinChooi** [comment]
Since a new PR [feat: add kpi and balanced board](https://github.com/omar-al-hendi/MyCovidView/pull/37) has been made so this PR I am closing this PR.

**Pull request closed without merging**